container/Dockerfile

```diff
@@ -37,19 +40,11 @@ RUN echo -e 'startup_message off \nhardstatus on \nhardstatus alwayslastline \nv
 # Setup vim 
 RUN echo -e 'syntax enable \nset tabstop=4 \nset number \nset hlsearch \nset incsearch' > ${HOME}/.vimrc
 
-# Setup spark/scala/python/rest
-ENV SPARK_HOME="/spark_home"
-ENV PATH=$PATH:$SPARK_HOME/bin:$SPARK_HOME/sbin
-RUN ${SETUP_AREA}/setup-java.sh
-RUN ${SETUP_AREA}/setup-spark.sh
-RUN ${SETUP_AREA}/setup-scala.sh
-RUN ${SETUP_AREA}/setup-python.sh
-RUN ${SETUP_AREA}/setup-kafka.sh
-RUN ${IOT_REPO_PATH}/rest/install.sh
-
-# Copy ES 2.0.1.0 client libraries
-RUN echo "Downloading Event Store 2.0.1.0 client libraries, it will take some time..."
-RUN wget -P ${SPARK_HOME}/jars/ https://repo1.maven.org/maven2/com/ibm/event/ibm-db2-eventstore-client-spark-2.4.6/2.0.1.0/ibm-db2-eventstore-client-spark-2.4.6-2.0.1.0.jar
+# copy json file to user_volume
+COPY ./es-releases.json /root/user_volume/
+
+# get variabls from json and setup sparke
```

Comment:
This comment seems wrong ?  Can you just change it to:
`# Setup all container dependencies based on the selected version`